src/snowcli/cli/common/cli_global_context.py

```diff
@@ -162,6 +163,9 @@ def output_format(self) -> OutputFormat:
     def set_output_format(self, value: OutputFormat):
         self._output_format = value
 
+    def set_project_definition(self, value: Dict):
```

Comment:
Can we move this setter (and connected property) to be placed under `set_experimental` to keep the same order as in the `init`?